docs/run_kat_tests.md

```diff
@@ -0,0 +1,35 @@
+
+# run-kat-tests
+
+This job installs all the required software to run
+[kube-app-testing](https://github.com/giantswarm/kube-app-testing).
```

Comment:
It took me a bit to realise the kind cluster was created by the script and not here.

Maybe some more info on the split between the job and the test script would help?